src/components/NotFound/NotFound.tsx

```diff
@@ -1,53 +1,44 @@
-import ListItem from '@material-ui/core/ListItem';
+import Box from '@material-ui/core/Box';
 import Typography from '@material-ui/core/Typography';
-import withWidth, { isWidthUp, WithWidthProps } from '@material-ui/core/withWidth';
+import styled from 'react-emotion';
 import React, { useCallback } from 'react';
-import { RouteComponentProps, withRouter } from 'react-router-dom';
+import { useHistory } from 'react-router-dom';
+
+import Button from '../../muiComponents/Button';
 
 import PackageImg from './img/package.svg';
-import { Card, EmptyPackage, Heading, Inner, List, Wrapper } from './styles';
 
-export const NOT_FOUND_TEXT = `Sorry, we couldn't find it...`;
-export const LABEL_NOT_FOUND = `The page you're looking for doesn't exist.`;
-export const LABEL_FOOTER_NOT_FOUND = 'Perhaps these links will help find what you are looking for:';
+export const NOT_FOUND_TEXT = "Sorry, we couldn't find it...";
+export const LABEL_NOT_FOUND = "The page you're looking for doesn't exist.";
 
-export type NotFoundProps = RouteComponentProps & WithWidthProps;
+const EmptyPackage = styled('img')({
+  width: '150px',
+  margin: '0 auto',
+});
 
-const HOME_LABEL = 'Home';
+const StyledHeading = styled(Typography)({
+  color: '#4b5e40',
+  marginBottom: 16,
```

Comment:
I think we have constants for this, right ? Otherwise we should create it.

Reply:
theme is coming...let's leave like this now 🙏